Ginger/GingerCoreNET/Run/RunListenerLib/CenteralizedExecutionLogger/AccountReportEntitiesDataMapping.cs

```diff
@@ -102,14 +107,17 @@ public static AccountReportAction MapActionEndData(GingerCore.Actions.Act action
             accountReportAction.ScreenShots = newScreenShotsList;
             
             accountReportAction.Artifacts = new List<AccountReport.Contracts.Helpers.DictObject>();
-          
-            foreach(ArtifactDetails artifact in action.Artifacts)
-            {                                   
-                string newArtifactPath = WorkSpace.Instance.RunsetExecutor.RunSetConfig.ExecutionID.ToString() + "/" +  Path.GetFileName(artifact.ArtifactNewPath);
-                newArtifactList.Add(artifact.ArtifactName, newArtifactPath);
-                accountReportAction.Artifacts.Add(new AccountReport.Contracts.Helpers.DictObject
-                { Key = artifact.ArtifactName, Value = newArtifactPath });                                 
-            }           
+
+            if(WorkSpace.Instance.Solution.LoggerConfigurations.UploadArtifactsToCentralizedReport == eUploadExecutionArtifactsToCentralizedReport.Yes)
+            {
+                foreach (ArtifactDetails artifact in action.Artifacts)
+                {
+                    string newArtifactPath = WorkSpace.Instance.RunsetExecutor.RunSetConfig.ExecutionID.ToString() + "/" +  Path.GetFileName(artifact.ArtifactReportStoragePath);
```

Comment:
Use path.DirectorySeparatorChar instead of /

Reply:
Done